code/game/gamemodes/shadowling/shadow_abilities.dm

```diff
@@ -0,0 +1,542 @@
+	panel = "Shadowling Abilities"
+	charge_max = 600
+	human_req = TRUE
+	clothes_req = FALSE
+	action_icon_state = "black_smoke"
+	action_icon = 'icons/mob/actions/actions_shadowling.dmi'
+	sound = 'sound/effects/bamf.ogg'
+
+/obj/effect/proc_holder/spell/blindness_smoke/cast(mob/living/carbon/human/user) //Extremely hacky
+	if(!shadowling_check(user))
+		revert_cast()
+		return
+	user.visible_message("<span class='warning'>[user] bends over and coughs out a cloud of black smoke!</span>")
+	to_chat(user, "<span class='shadowling'>You regurgitate a vast cloud of blinding smoke.</span>")
+	var/obj/item/reagent_containers/glass/beaker/large/B = new /obj/item/reagent_containers/glass/beaker/large(user.loc) //hacky
+	B.reagents.clear_reagents() //Just in case!
+	B.invisibility = INFINITY //This ought to do the trick
+	B.reagents.add_reagent("blindness_smoke", 10)
+	var/datum/effect_system/smoke_spread/chem/S = new
+	S.attach(B)
+	if(S)
+		S.set_up(B.reagents, 4, 0, B.loc)
+		S.start()
+	qdel(B)
+
+
+
+
+/////////////////////
+////SONIC SCREECH////
+/////////////////////
+
+/obj/effect/proc_holder/spell/unearthly_screech //Damages nearby windows, confuses nearby carbons, and outright stuns silly cones
+	name = "Sonic Screech"
+	desc = "Deafens, stuns, and confuses nearby people. Also shatters windows."
+	panel = "Shadowling Abilities"
+	charge_max = 300
+	human_req = TRUE
+	clothes_req = FALSE
+	action_icon_state = "screech"
+	action_icon = 'icons/mob/actions/actions_shadowling.dmi'
+	sound = 'sound/effects/screech.ogg'
+
+/obj/effect/proc_holder/spell/unearthly_screech/cast(list/targets,mob/user = usr)
+	if(!shadowling_check(user))
+		revert_cast()
+		return
+	user.audible_message("<span class='warning'><b>[user] lets out a horrible scream!</b></span>")
+	for(var/turf/T in view(7, src))
+		for(var/mob/target in T.contents)
+			if(is_shadow_or_thrall(target))
+				if(target == user) //No message for the user, of course
+					continue
+				else
+					continue
+			if(iscarbon(target))
+				var/mob/living/carbon/M = target
+				to_chat(M, "<span class='danger'><b>A spike of pain drives into your head and scrambles your thoughts!</b></span>")
+				M.confused += 10
+				M.adjustEarDamage(0, 30)//as bad as a changeling shriek
+			else if(issilicon(target))
+				var/mob/living/silicon/S = target
+				to_chat(S, "<span class='warning'><b>ERROR $!(@ ERROR )#^! SENSORY OVERLOAD \[$(!@#</b></span>")
+				playsound(S, 'sound/machines/warning-buzzer.ogg', 50, 1)
+				var/datum/effect_system/spark_spread/sp = new /datum/effect_system/spark_spread
+				sp.set_up(5, 1, S)
+				sp.start()
+				S.Knockdown(6)
+		for(var/obj/structure/window/W in T.contents)
+			W.take_damage(rand(80, 100))
+
+
+/////////////////////
+////SHUTTLE DELAY////
+/////////////////////
+
+/obj/effect/proc_holder/spell/targeted/shadowling_extend_shuttle
+	name = "Destroy Engines"
+	desc = "Extends the time of the emergency shuttle's arrival by fifteen minutes. This can only be used once."
```

Comment:
This was a horrible terrible ability that served as a band-aid fix to the larger problem of shuttle calls. Shuttle calls were the only way to combat the snowballing that shadowlings had with thralls, and due to shadowling's nature it still is. Nightmares circumvented this because the only way to put out lights is to hit someone with the light eater, making it more viable to attack with lights, but due to shadowlings having so many escape abilities they're very difficult to actually kill or pin down. 

Making it more feasible to actually hunt down the shadowlings, as well as making deconversion less tedious, is a far preferable solution; this is a badly-made ability that only serves to frustrate everyone involved: shadowlings are frustrated because they only put off the shuttle, and crew are frustrated because they have no way to fight back against it.